next/src/components/_global/offersComponent/offersComponent.tsx

```diff
@@ -0,0 +1,95 @@
+import styles from './offersComponent.module.scss';
+import Heading from '@/components/ui/heading/Heading';
+import { type Offers } from '@/types/_global/Offers';
+import Markdown from '@/components/ui/Markdown';
+import Button from '@/components/ui/button/Button';
+
+export default function OffersComponent({ data: { offers } }: { data: Offers }) {
+  return (
+    <section className={`${styles.offersComponent} fullWidthBackground`}>
+      <div className={styles.offers}>
+        {offers.map(({ addition, cta, description, heading, price, title }, i) => (
+          <div
+            key={i}
+            className={styles.offer}
+          >
+            <Heading
+              type='h2'
+              className={styles.heading}
+            >
+              {heading}
+            </Heading>
+            <div className={styles.content}>
+              <Markdown.h3 className={styles.title}>{title}</Markdown.h3>
+              <p className={styles.price}>{price}</p>
+            </div>
+            <Markdown className={styles.list}>{description}</Markdown>
+            <div className={styles.additionWrapper}>
+              {RoundedPlusIcon()}
```

Comment:
Podmień proszę na "< RoundedPlusIcon />"

To jest component, nie musisz używać go jako funkcję.